core/OneCloud.py

```diff
@@ -42,10 +42,23 @@ def req(self, url, method = "get", data=None):
 
 
     def server(self, action:str = "list", template:str = ""):
-        def get_list():
-            return self.req(url=self._base_url + "server", method="get")
+        """
+        Method working with 1cloud api, using req() to create, deletea and update servers.
+        :: action:str = list, create, delete, update.
+        ::template = server config template.
+        """
+        def get_list() -> list[dict[str,str]]:
+            """
+            Function returns the list of created servers on 1cloud prod.
+            Function calling req(). 
+            """
+            servers_list = self.req(url=self._base_url + "server", method="get")
+            return servers_list
         
-        def create():
+        def create() -> list[dict[str,str]]:
+            """
+            Function create the servers and returns list of panding servers without credentions to connecting Ansible.
+            """
             serv_in_temp = self.serv.server_parser(template_name=template) #List of servers in template
```

Comment:
template is implicitly captured in this closure. I'd rather pass it explicitly as a parameter, so it's obvious and later if we need to separate this code, we don't have to think what this code captures from outside